schema/codelists/relatedProcess.csv

```diff
@@ -1,9 +1,10 @@
 Code,Title,Description
-framework,Framework agreement procedure first stage,This contracting process follows on from the related process to establish a framework. The suppliers on the framework are listed in the award section of the related process.
+framework,Framework agreement procedure first stage,This contracting process follows on from the related process that established a framework agreement. The suppliers on the framework are listed in the award section of the related process.
 planning,Planning process,This contracting process follows on from the related planning process.
 parent,Parent contract (for sub-contracts),This contracting process might result in a sub-contract of the related process.
-prior,Prior process,This contracting process is the renewal or replacement of the related prior process.
-unsuccessfulProcess,Unsuccessful process,This contracting process follows on from an previous unsuccessful process.
-subContract,Sub-contract,The related process might result in a sub-contract of this contract.
-replacementProcess,Replacement process,The related process might result in the replacement of this contract.
-renewalProcess,Renewal process,The related process might result in the renewal of this contract.
+prior,Prior process,This contracting process follows on from the related successful process.
```

Comment:
```suggestion
prior,Prior process,The related process is a contracting process that succeeded in implementing related contracts.
```

Reply:
I would've expected this description to align with the description of the 'unsuccessful' code in the contract status codelist, which is about whether the contract is successfully concluded rather than successfully implemented.

Reply:
We can change it to "concluded".

I used "implemented" in the meaning of the definition of "contracting process"

> All the actions aimed at implementing one or more contracts.

That said, in this context, "implemented" is less clear.

Reply:
Done in 1be3af934fe6ca688f387aa3777497f96220bae1